sdk/harambe/observer.py

```diff
@@ -32,6 +31,12 @@ class DownloadMeta(TypedDict):
     filename: str
 
 
+class HTMLMetadata(TypedDict):
+    download: DownloadMeta
+    html: str
+    inner_text: str
```

Comment:
Thoughts on this API for the sdk function? Can also just have the DownloadMeta fields at the top level (maybe that's better)

Reply:
ended up going with that